examples/python/FullScriptSupport_Pandas/README.md

```diff
@@ -0,0 +1,45 @@
+# Example: Full script support using Pandas
+This example plugin includes support for all script functionality and is based on the original [Full Script Support](../FullScriptSupport/README.md) Python example. The implementation of this plugin differentiates mainly by the use of the Pandas library, the data received from Qlik is now saved to a Pandas data frame. Another difference is that we now use the `exec` method to evaluate the script rather than the `eval` method, as we did in the original example plugin. This change opens up the possibility to for instance pass a multiline script from Qlik.
```

Comment:
The implementation of this plugin differs mainly in the use of the Pandas library. In addition, the data received from Qlik is now saved to a Pandas data frame. In this example, we use the `exec` method to evaluate the script rather than the `eval` method, as we did in the original example plugin. This change makes it possible to pass a multiline script from Qlik.